README.md

```diff
@@ -20,7 +19,21 @@
   - [Documentation](#documentation)
 
 # OpenShift Hive
-API driven OpenShift cluster provisioning and management
+API driven OpenShift 4 cluster provisioning and management.
+
+Hive is an operator which runs as a service on top of OpenShift.
+The Hive service can be used to provision and manage OpenShift clusters.
+There is no limit on how many cluster Hive can provision or manage.
```

Comment:
IMO one sentence per line for documentation is a good practice which helps in long term maintainability of the docs. Refer https://rhodesmill.org/brandon/2012/one-sentence-per-line/

Reply:
I'm not sure this last sentence is a good idea, there is almost surely a limit, we just haven't tried to find out where it is yet.

Reply:
Could you fit in somewhere that hive's responsibilities are API driven provisioning, day 2 configuration management, reshaping, and deprovisioning.

Reply:
> I'm not sure this last sentence is a good idea, there is almost surely a limit, we just haven't tried to find out where it is yet.

Agree. I will move this sentence to FAQ.